.travis.yml

```diff
@@ -11,14 +12,21 @@ env:
     - JRUBY_OPTS="$JRUBY_OPTS --debug"
 
 rvm:
-  - jruby-9.1.6.0
   - 2.0.0
   - 2.1
   - 2.2
   - 2.3.0
+  - 2.3.1
+  - 2.3.2
+  - 2.3.3
   - 2.4.0
+  - ruby-head
+  - jruby-9.1.6.0
+  - jruby-9.1.7.0
```

Comment:
If you put jruby at the top it will start running first... which is a good thing as it takes the longest